src/pages/home/HomePage.tsx

```
@@ -26,53 +28,76 @@ interface HomeExpenseItemType {
+    </Flex>
   );
+}
+
+function MainHeader() {
   const navigate = useNavigate();
   const theme = useTheme();
+  return (
+    <Header
+        type="TitleCenter"
+        leftButtonContent={
+          <Button variant="text" onClick={() => navigate(ROUTE.login)}>
```

ESLint

[error] Delete `··`

(prettier/prettier)